frontends/web/src/components/transactions/transactions.jsx

```diff
@@ -47,8 +46,8 @@ export default class Transactions extends Component {
                     <div className={style.date}>{t('transaction.details.date')}</div>
                     <div className={style.address}>{t('transaction.details.address')}</div>
                     <div className={style.status}>{t('transaction.details.status')}</div>
-                    <div className={style.fiat}>{t('transaction.details.fiat')}</div>
-                    <div className={style.currency}>{unit}</div>
+                    <div className={style.fiat}>{t('transaction.details.fiat')} {t('transaction.details.amount')}</div>
```

Comment:
IMO don't combine the 2 translations, just use `{t('transaction.details.fiat')}` and update the i18n

Reply:
:+1: being smart about combining two i18n entries often just does not work in one of the languages due to different grammar, structure etc.

Reply:
I see, gotcha 👍